patches/release-plan@0.9.0.patch

```diff
@@ -1,30 +0,0 @@
-diff --git a/dist/plan.js b/dist/plan.js
-index 3e1641cb2cca0ec94e310b5b25bb98b78001c3cf..d3f2396943d01d3a7208b77ae4e9c0f7ee70eea7 100644
---- a/dist/plan.js
-+++ b/dist/plan.js
-@@ -39,7 +39,7 @@ class Plan {
-                 solution.set(pkgName, { impact: undefined, oldVersion: entry.version });
-             }
-             else {
--                const newVersion = inc(entry.version, impact);
-+                const newVersion = inc(entry.version, this.#configureImpact(pkgName, impact));
-                 solution.set(pkgName, {
-                     impact,
-                     oldVersion: entry.version,
-@@ -51,6 +51,16 @@ class Plan {
-         }
-         return solution;
-     }
-+    #configureImpact(pkgName, impact) {
-+        const packageJson = this.#pkgs.get(pkgName)?.pkg;
-+        if (packageJson && packageJson['release-plan']?.semverIncrementAs) {
-+            const semverOverrides = packageJson['release-plan'].semverIncrementAs;
```

Comment:
@mansona could the new version handle pre-release versions? any setup we can/need to update?

Reply:
the patch got upstreamed and the config is the same

Reply:
yes @void-mAlex is right, the fix has been upstreamed and you will see the effect once this is merged i.e. you will see the Plan release PR before anything will get released and it will say what version it's about to release 👍